druid-shell/src/window.rs

```diff
@@ -123,6 +123,26 @@ impl WindowHandle {
         self.0.show_titlebar(show_titlebar)
     }
 
+    /// Set the position of the window
```

Comment:
What are the units? And I'm guessing this will set the position of the top-left of the window?